features/works/work_drafts.feature

```diff
@@ -187,3 +187,47 @@ Feature: Work Drafts
       When I follow "Next Chapter"
         And I press "Post Chapter"
       Then I should see "Words:16"
+
+  Scenario: A work draft should not have comments
+  Given I am logged in as "Scott" with password "password"
+    And the draft "scotts commentless draft"
+  When I am on scott's works page
+    Then I should see "Drafts (1)"
+  When I follow "Drafts (1)"
+    Then I should see "scotts commentless draft"
+      And I should not see "Comments"
```

Comment:
These tests seem to be checking the drafts index page for comment buttons instead of the work page?
Assuming that's unintentional, the tests can be tightened up a bit by using the step `I view the work "scotts chaptered draft"` rather than manually navigating to the work page.